src/backend/joseki.be/webapp/Handlers/GetOverviewDetailsHandler.cs

```diff
@@ -0,0 +1,486 @@
+                        .ToArray());
+
+            results.TryAdd("collection", allChecks.GroupBy(x => x.Collection.Type + " " + x.Collection.Name)
+                        .Select(x => new
+                        {
+                            type = x.First().Collection.Type,
+                            name = x.First().Collection.Name,
+                            str = x.First().Collection.Type + ":" + x.First().Collection.Name,
+                        })
+                        .OrderBy(x => x.str)
+                        .Select(filter =>
+                            new CheckFilter(
+                                filter.str,
+                                !filteredChecks.Any(c => c.Collection.Type == filter.type && c.Collection.Name == filter.name)))
+                        .ToArray());
+
+            results.TryAdd("control", allChecks.GroupBy(x => x.Control.Id)
+                        .Select(x => x.First().Control.Id)
+                        .OrderBy(x => x)
+                        .Select(filter => new CheckFilter(filter, !filteredChecks.Any(c => c.Control.Id == filter)))
+                        .ToArray());
+
+            results.TryAdd("resource", allChecks.GroupBy(x => x.Resource.Type + " " + x.Resource.Name)
+                        .Select(x => new
+                        {
+                            type = x.First().Resource.Type,
+                            name = x.First().Resource.Name,
+                            str = x.First().Resource.Type + ":" + x.First().Resource.Name,
+                        })
+                        .OrderBy(x => x.str)
+                        .Select(filter =>
+                            new CheckFilter(
+                                filter.str,
+                                !filteredChecks.Any(c => c.Resource.Type == filter.type && c.Resource.Name == filter.name)))
+                        .ToArray());
+
+            results.TryAdd("result", allChecks.GroupBy(x => x.Result.ToString())
+                        .Select(x => x.First().Result.ToString())
+                        .OrderBy(x => x)
+                        .Select(filter => new CheckFilter(filter, !filteredChecks.Any(c => c.Result.ToString() == filter)))
+                        .ToArray());
+
+            return results;
+        }
+
+        /// <summary>
+        /// Returns autocomplete data for selected field.
+        /// </summary>
+        /// <param name="date">The date to get details for.</param>
+        /// <returns>string array.</returns>
+        public async Task<List<OverviewCheck>> GetChecks(DateTime date)
+        {
+            // 1. get audit which represents the "date".
+            var audits = await this.db.Set<AuditEntity>()
+                .Include(i => i.InfrastructureComponent)
+                .AsNoTracking()
+                .Where(i => i.Date.Date == date.Date)
+                .OrderByDescending(i => i.Date)
+                .ToArrayAsync();
```

Comment:
Unfortunately, this request might return you a kind of _duplicates_.
- each day from each scanner we can receive `0..N` audits;
- at the moment, all these data is just saved in the same database table;
- on the user interface, we should display only the most recent audit for each single day.

It means, the method should do two things:
- get all the audits for a given day from the db
- group them by `ComponentId` and take only the latest one

Take a look on [this method](https://github.com/deepnetworkgmbh/joseki/blob/master/src/backend/joseki.be/webapp/Database/InfraScoreDbWrapper.cs#L74-L90). Likely, that's exactly what you need here